### ISSUE TITLE
game load crash

### ISSUE THREAD

**Issue by msphell** [Norbyte/ositools#119]
### Operating System

Windows

### Extender Version

v58

### Game Version

v3.6.117.3735

### Bug Summary

game crash

### Links

![image](https://user-images.githubusercontent.com/1471927/222903758-17b2cf0b-99a3-4250-8c79-1396388d3e0b.png)

**Comment by charlesbcqq** [comment]
I got the same exact issue. The game was running smoothly at v57, but when the script extender updated to v58 earlier today, the same exact error happened to me. I can't even get to the game's home menu. I just hope it just stayed at v57. I hope this gets fixed soon. I really loved playing the games with the mods.

**Comment by charlesbcqq** [comment]
Hey, I fixed it! Just do the following:
1.) Re-install the script extender by downloading the dxgi.dll again.
2.) Create a text document and paste this code:

> {
    "UpdateChannel": "Devel",
    "TargetVersion": "57.0.0.0",
    "TargetResourceDigest": "2867ca7912b02b4e898ad9fff2e45e7d1eb1dc7932512f8633bfbf3ba69d3682",
    "Debug": true
}

3.) Rename the text document to criptExtenderUpdaterConfig.json
4.) Relaunch the game and it should now be back to V57 and won't auto-update anymore. 

I hope that helped. I don't even know how to code. I just found out how to turn off the auto-update.

**Comment by cepreu2github** [comment]
I confirm [charlesbcqq](https://github.com/charlesbcqq) solution as working. Probably should be in installation instructions somewhere since now it is only way to get game up and running.

**Comment by msphell** [comment]
> Hey, I fixed it! Just do the following: 1.) Re-install the script extender by downloading the dxgi.dll again and putting it in the same bin folder. 2.) Create a text document in the same bin folder and paste this code:
> 
> > {
> > "UpdateChannel": "Devel",
> > "TargetVersion": "57.0.0.0",
> > "TargetResourceDigest": "2867ca7912b02b4e898ad9fff2e45e7d1eb1dc7932512f8633bfbf3ba69d3682",
> > "Debug": true
> > }
> 
> 3.) Rename the text document to ScriptExtenderUpdaterConfig.json 4.) Relaunch the game and it should now be back to V57 and won't auto-update anymore.
> 
> I hope that helped. I don't even know how to code. I just found out how to turn off the auto-update.

**Comment by LaughingLeader** [comment]
I'm curious what mods are active in the load order here, since v58 has been in the "Development" branch for a while, and we haven't had any crashes in the main menu (as far as I know).

A copy of the modsettings.lsx would be helpful (this contains the mod order). You can find that file here:
```
%USERPROFILE%\Documents\Larian Studios\Divinity Original Sin 2\PlayerProfiles\ProfileName\modsettings.lsx
```

**Comment by LaughingLeader** [comment]
This should be fixed in the latest update: d756ad1b6e6ac1123e51f91a4aa73bd53b6fba54

**Comment by cepreu2github** [comment]
[modsettings.zip](https://github.com/Norbyte/ositools/files/10890752/modsettings.zip)

In case if it still needed.
